### PR TITLE
React 16 + Hapi 17

### LINKED ISSUE #71: Update to react v16
I just updated the [vision](https://github.com/hapijs/vision) examples -- one of which uses this plugin for [examples/jsx](https://github.com/hapijs/vision/blob/master/examples/jsx/index.js#L7). I've got react 16 as a dependency and I'm running into `hapi-react-views@9.3.0 requires a peer of react@15.x.x`

If you want, I'd be willing to submit a PR

### PR DESCRIPTION
**Pull request by jedireza**
Resolves #71

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,10 +112,7 @@ const renderOpts = {
     }
 };
 
-server.render('template', context, renderOpts, (err, output) => {
-
-    // ...
-});
+const output = await server.render('template', context, renderOpts);
```
Comment (jedireza) [inline README.md]: @wswoodruff should/could we assume `server.render` will become `async` also?

Comment (wswoodruff) [reply README.md]: Yup! 👍

File: examples/layout-component/server.js [hunk 2]
```diff
@@ -5,17 +5,17 @@ const HapiReactViews = require('../..');
...
 
-const server = new Hapi.Server();
-server.connection({ port: process.env.PORT });
-server.register(Vision, (err) => {
+const main = async function () {
 
-    if (err) {
-        console.log('Failed to load vision.');
-    }
+    const server = new Hapi.Server({
```
Comment (wswoodruff) [inline examples/layout-component/server.js]: no more `new `

Comment (jedireza) [reply examples/layout-component/server.js]: Thanks. I updated all instances.

**Pull request merged**